src/CommandWrapper.tsx

```diff
@@ -48,7 +48,7 @@ export const CommandWrapper: FC<
             initial={{ opacity: 0, y: 20 }}
             animate={{ opacity: 1, y: 0, scale: animate ? 0.98 : 1 }}
             exit={{ opacity: 0, y: 20 }}
-            transition={{ duration: 0.1 }}
+            transition={{ duration: config?.animationDuration || 0.1 }}
```

Comment:
I think a `??` would be better here for cases where the animation needs to be instant! 🙂

Reply:
done!